openssl-dynamic/src/main/c/ssl.c

```diff
@@ -51,6 +51,8 @@ struct CRYPTO_dynlock_value {
 };
 
 
+#define ERR_LEN 256
```

Comment:
lets make this more generally available (in a shared header file) and use it everywhere ... unless we have good reason not to. `s/ERR_error_string/ERR_error_string_n`